java/src/jmri/jmrix/openlcb/OlcbEventNameStore.java

```diff
@@ -20,28 +20,52 @@ public OlcbEventNameStore() {
     
     /**
      * @param eventID The EventID being searched for
-     * @return The name associated with that EventID or null
+     * @return The name associated with that EventID or the event ID in dotted hex
      */
     public String getEventName(EventID eventID) {
         var tag = tagmgr.getBySystemName(OlcbConstants.tagPrefix+eventID.toShortString());
-        if (tag == null) return null;
+        if (tag == null) return eventID.toShortString();
         var name = tag.getUserName();
-        if (name == null || name.isEmpty()) return null;
+        if (name == null || name.isEmpty()) return eventID.toShortString();
         return name;
     }
     
+    /**
+     * @param eventID The EventID being searched for
+     * @return true if there is an associated name
+     */
+    public boolean hasEventName(EventID eventID) {
+        var tag = tagmgr.getBySystemName(OlcbConstants.tagPrefix+eventID.toShortString());
```

Comment:
No need to change and a personal preference that you can ignore. But in my eyes the code would be easier to read if there is a space before and after the + sign.

OlcbConstants.tagPrefix + eventID.toShortString()